### PR TITLE
SDCICD-20. Expand scale tests.

### PR DESCRIPTION
**Pull request by meowfaceman**
More of the openshift-scale workloads are now run against osde2e
clusters during scale testing. This also plumbs through pbench SSH
credentials through to the scale tests so that they can publish pbench
results to the scale pbench server.

Dependent on https://github.com/openshift/release/pull/7262.

### CODE REVIEW

**Comment by meowfaceman** [comment]
/hold
This is going to be depend on https://github.com/openshift-scale/workloads/pull/115 as well.

**Comment by meowfaceman** [comment]
/hold cancel
https://github.com/openshift-scale/workloads/pull/115 was merged.

File: pkg/e2e/scale/nodes_and_pods.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	ginkgo.It("should be tested with NodeVertical", func() {
+		// setup runner
+		scaleCfg := scaleRunnerConfig{
+			Name:         "node-vertical",
+			PlaybookPath: "workloads/nodevertical.yml",
+		}
+		r := scaleCfg.Runner(h)
+
+		// only test on 3 nodes
+		numNodes := 3
```
Comment (jeefy) [inline pkg/e2e/scale/nodes_and_pods.go]: Nit: Should this be a const?

Comment (meowfaceman) [reply pkg/e2e/scale/nodes_and_pods.go]: Good idea. Will make that change.

**Pull request merged**